modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -519,10 +522,8 @@ class NDB_Form_Imaging_Browser extends NDB_Form
     function _updateVisitStatus()
     {
         if (!empty($_POST['visit_status'])) {
-            $save_visit_status
```

Comment:
@driusan @jstirling91 

1) So I have been looking into a potential answer, here is one logical thing I can think of, but it is a pure guess:
These status eventually get concatenated in a `$message` that gets written to the `notification_spool` table in the variable called `$not_status` on line 577. I am guessing changing `' '` to `Unrated` makes the message being spooled (and which we have the option to email in an automated way) more meaningful;
Here is an example message:
`777202 / scanner test_phantom_OTT_20170802 
                    - MRI QC status changed to Pending  Unrated false`
Pending, Unrated, and False are the respective status of: visit_pending, visit_status, and visit_caveat fields.

Perhaps this is the real reason for this change. These are my two cents!

2) On another note, line 382 has a value of `unrated` with a small `u` that also gets assigned to `' '`. I wonder if this also has to be added to the Databse.class.inc!